### ISSUE TITLE
Deprecation message autowiring alias ContainerInterface

### ISSUE THREAD

**Issue by ToshY** [MacPaw/symfony-health-check-bundle#24]
Using Symfony 5.3 the profiler shows the following deprecation message:

```
Since symfony/dependency-injection 5.1: The "Symfony\Component\DependencyInjection\ContainerInterface" autowiring alias is deprecated. Define it explicitly in your app if you want to keep using it. It is being referenced by the "symfony_health_check.doctrine_check" service.
Show context    Hide trace
{▼
  /app/vendor/symfony/dependency-injection/Compiler/ResolveReferencesToAliasesPass.php:68 {▼
    Symfony\Component\DependencyInjection\Compiler\ResolveReferencesToAliasesPass->getDefinitionId(string $id, ContainerBuilder $container): string …
    ›     $deprecation = $alias->getDeprecation($id);
    ›     trigger_deprecation($deprecation['package'], $deprecation['version'], rtrim($deprecation['message'], '. ').'. It is being referenced by the "%s" '.($container->hasDefinition($this->currentId) ? 'service.' : 'alias.'), $this->currentId);
    › }
  }
  /app/vendor/symfony/dependency-injection/Compiler/ResolveReferencesToAliasesPass.php:51 {▼
    › 
    › $defId = $this->getDefinitionId($id = (string) $value, $this->container);
    › 
  }
}
```

**Comment by Yozhef** [comment]
@ToshY Thank you for your appeal, we will fix it soon

**Comment by Yozhef** [comment]
@ToshY fix in develop - but we have we have problems with automatic releases and will deal with them soon

**Comment by Yozhef** [comment]
@ToshY we have released release 0.4.0 it is ready to use, thanks for contacting.
